components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/JWTCacheEntry.java

```diff
@@ -0,0 +1,43 @@
+
+/*
+ * Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
license header need to be corrected to carry just the year, since this is newly adding file.